assets/lokomotive-kubernetes/packet/flatcar-linux/kubernetes/calico-host-protection/templates/host-endpoints.yaml

```diff
@@ -10,5 +10,5 @@ metadata:
     {{- end }}
 spec:
```

Comment:
Not sure why we have this file in the first place, either. The calico-hostendpoint-controller creates this for workers, right?

Reply:
It was there for a long time. Perhaps it can be removed now so controller will take care of everything?

Reply:
For workers it was removed for a long time. I think I follow the code (sorry, next time will see it locally) and is only for controllers, so :ok_hand:

Reply:
Do you think controller could also create the objects for controller nodes? We have boostrap firewall policies, so I think it should be fine?

Reply:
I don't think I'd change who creates what, just wanted to make sure this was only for controllers.

I'd rather wait on that and see how to use the new coming feature in calico to replace the controller (if possible, if not maybe create another PR for calico adding what we miss :))